### ISSUE TITLE
Accessibility issue on show password button/icon

### ISSUE THREAD

**Issue by Matthewundy** [okta/okta-signin-widget#1145]
### I'm submitting a
  - [x ] bug report
  - [ ] feature request

### Background info

Accessibility issue affecting screen readers

### What went wrong?

The “Show password” button is empty. The aria-label technique is not a good one if there is nothing in the button when CSS is deactivated. 
a. Remove the aria-label attribute from the <button> 
b. Add a screen reader only text inside it: 
<button class="c-pwdtoggle__btn" type="button" dataaction="togglePasswordVisibility">     <span class="c-pwdtoggle__icon" aria-hidden="true"></span>     <span class="sr-only">Show password</span> </button> 
3. Also, blind people using assistive technologies need to know what the state of the button is (a visual information can’t be sufficient because it can’t be read). When the “Show password” button is active, change its name for “Hide password”: 
<button class="c-pwdtoggle__btn" type="button" dataaction="togglePasswordVisibility">     <span class="c-pwdtoggle__icon" aria-hidden="true"></span>     <span class="sr-only">Hide password</span> </button> 

### Steps to reproduce

Standard widget HTML

### Your environment

  - Okta Sign-In Widget Version: Any
  - Browser: Any
  - OS: Any
  - Language: Any

**Comment by swiftone** [comment]
Internal ref: OKTA-293145

**Comment by dexraven** [comment]
I am also seeing this issue as well. Is there any fix out for this?

**Comment by jchabotamica** [comment]
You also cannot "TAB" to the "button" so its not keyboard accessible as well.  If you make it a button in the html, it should be keyboard accessible

**Comment by aarongranick-okta** [comment]
Have you enabled `features.showPasswordToggleOnSignInPage` ?

https://github.com/okta/okta-signin-widget#feature-flags

**Comment by jchabotamica** [comment]
@aarongranick-okta  yes - that is how we are having it show.  The problem is that it fails accessibility (ie. screen readers, keyboard only users, etc.)

**Comment by mlpeace** [comment]
We also have noticed this bug still exists. Is there an ETA on when this will be fixed, please?

**Comment by kk185256** [comment]
is there any work around for this?
